KeywordSearch/ivy.xml

```diff
@@ -20,7 +20,7 @@
     <dependencies>
         <!-- Solr -->
         <dependency conf="solr-libs->default" org="org.apache.solr" name="solr-cell" rev="4.9.1"/>
-        <dependency conf="solr-war->default" org="org.apache.solr" name="solr" rev="4.9.1"/> <!-- the war file -->
+        <dependency conf="solr-war->default" org="org.apache.solr" name="solr" rev="4.9.1" transitive="false" /> <!-- the war file -->
```

Comment:
I am really surprised this was needed.  Shouldn't setting transitive to false, result in Ivy pulling down less stuff?